src/components/AlbumList.jsx

```diff
@@ -1,12 +1,25 @@
 import AlbumCard from "./AlbumCard";
-import "./AlbumList.css";
+import "./css/AlbumList.css";
 
-export default function AlbumList({ data, title, className }) {
+export default function AlbumList({ data, title, onToggleSave, onSavedCheck }) {
   return (
-    <div className={className}>
+    <div className="album-list">
       <h2 className="album-list-title">{title}</h2>
       <ul>
-        {data.length > 0 && data.map((item) => <li key={item.id}>{<AlbumCard data={item} />}</li>)}
+        {data.length > 0 &&
+          data.map((item) => (
+            <li key={item.id}>
+              {
+                <AlbumCard
+                  data={item}
+                  isSaved={onSavedCheck(item.id)}
+                  onToggleSave={() => {
+                    onToggleSave(item.id);
+                  }}
```

Comment:
We can shorten this slightly...
```suggestion
                  onToggleSave={() => onToggleSave(item.id)}
```